tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_bluetooth.py

```diff
@@ -27,15 +23,25 @@ def test_toggle_bluetooth_settings(self):
         settings = Settings(self.marionette)
         settings.launch()
 
+        self.assertFalse(self.data_layer.bluetooth_is_enabled)
+        self.assertFalse(self.data_layer.get_setting('bluetooth.visible'))
+
         bluetooth_settings = settings.open_bluetooth_settings()
         bluetooth_settings.enable_bluetooth()
 
+        self.assertTrue(self.data_layer.bluetooth_is_enabled)
+        self.assertTrue(self.data_layer.get_setting('bluetooth.visible'))
+
         bluetooth_settings.tap_rename_my_device()
         bluetooth_settings.type_phone_name(device_name)
         bluetooth_settings.tap_update_device_name_ok()
 
         bluetooth_settings.enable_visible_to_all()
```

Comment:
That check is now done with the self.assertTrue(self.data_layer.get_setting('bluetooth.visible')) call.